engine/src/main/java/org/terasology/rendering/opengl/DefaultDynamicFBOs.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.opengl;
+
+import java.nio.ByteBuffer;
+import org.terasology.assets.ResourceUrn;
+import static org.terasology.rendering.opengl.ScalingFactors.FULL_SCALE;
+
+/**
+ * TODO: Add javadocs
+ */
+public enum DefaultDynamicFBOs {
```

Comment:
Please remind me. What's the usefulness of having them as enums rather than standard FBOs instantiated when the DisplayResolutionDependentFBOs gets instantiated or initialized?